members/CusD7a2f17071574702549.yaml

```diff
@@ -0,0 +1,39 @@
+bioguide: CusD7a2f17071574702549
+contact_form:
+  method: post
+  steps:
+    - visit: "https://ngpvan-freedom-tools.herokuapp.com/federalRegister"
+    - find:
+        - selector: input#document_id
+    - fill_in:
+        - name: Document Id
+          selector: "#document_id"
+          value: '2019-24366'
+          required: true
+        - name: Comment
+          selector: "#comment"
+          value: $MESSAGE
+          required: true
+        - name: First Name
+          selector: "#firstname"
+          value: $NAME_FIRST
+          required: false
+        - name: Last Name
+          selector: "#lastname"
+          value: $NAME_LAST
+          required: false
+        - name: zip5
+          selector: "#zip"
+          value: $ADDRESS_ZIP5
+          required: false
+        - name: email
+          selector: "#email"
+          value: $EMAIL
+          required: false
```

Comment:
do these `required` values match the actual form?

Reply:
yes. And i just committed an updated with two steps that are in the form but missing from the yaml (both `required: false`)